src/current/v22.2/orchestrate-cockroachdb-with-kubernetes-multi-cluster.md

```diff
@@ -65,7 +65,7 @@ To enable the pods to communicate across regions, we peer the VPCs in all 3 regi
 {{site.data.alerts.callout_danger}}
 Until December 2019, Google Cloud Platform restricted clients to using a load balancer in the same region. In the approach documented below, the DNS servers from each Kubernetes cluster are hooked together by exposing them via a load balanced IP address that is visible to the public Internet. None of the services in your Kubernetes cluster will be accessible publicly, but their names could leak out to a motivated attacker.
 
-You can now [enable global access](https://cloud.google.com/load-balancing/docs/internal/setting-up-internal#ilb-global-access) on Google Cloud Platform to allow clients from one region to use an internal load balancer in another. We will update this tutorial accordingly.
+You can now [enable global access](https://cloud.google.com/load-balancing/docs/internal/setting-up-internal#ilb-global-access) on Google Cloud Platform to allow clients from one region to use an internal load balancer in another, instead of using this approach.
```

Comment:
I had to read this a couple of times because I was confused as to whether "instead of using this approach" referred to the current paragraph. What about the suggestion? 

Or, maybe you could flip it and have "Instead of using this approach, you can now enable global access...." I think starting this paragraph make it feels more connected to the prior paragraph.
```suggestion
You can now [enable global access](https://cloud.google.com/load-balancing/docs/internal/setting-up-internal#ilb-global-access) on Google Cloud Platform to allow clients from one region to use an internal load balancer in another, instead of using the following approach.
```

Reply:
Thinking about it, I just think that "this" needs to be defined in that part of the sentence to confirm we're talking about the documented approach on the page.

Reply:
Good point. I reworked the note a bit more than. that, and I think it's a lot better.